### ISSUE TITLE
Print test results to log file

### ISSUE THREAD

**Issue by AndCook** [AndCook/selenium_testing#3]
Currently the script just prints the results of the test to stdout. We really want instead the results to be printed to some sort of log file. This involves some design decisions that need to be made, but it really shouldn't be that bad. Working with files and stuff is pretty easy in python, and we're not the first team to want to print test results to log files.

**Comment by AndCook** [comment]
I'm expanding this issue, because I realized that more work than just printing to a log file should be done. Right now the creation of the errors is too unstructured. There should be some better defined way of compiling the information for the log file than building strings with newlines and tabs. I think there must be some way to create this structure using the Tester class in the new util file.

Basically, at some point we should move beyond just passing strings around for errors.
